platforms/access-bridge/src/services/access-service.ts

```diff
@@ -0,0 +1,118 @@
+import jwt from 'jsonwebtoken';
+import {
+  AccessTokensResponse,
+  GenerateAccessTokensParams,
+  RefreshAccessTokensParams,
+} from '@jwp/ott-common/types/access.js';
+
+import { ACCESS_CONTROL_CLIENT, API_SECRET } from '../app-config.js';
+import { BadRequestError, ForbiddenError, NotFoundError, ParameterInvalidError, isJWError } from '../errors.js';
+import { put } from '../http.js';
+
+/**
+ * AccessService handles interactions with the access management APIs.
+ * It provides methods to generate and refresh access tokens.
+ */
+export class AccessService {
+  private apiSecret: string;
+  private accessControlClient: string;
+
+  constructor() {
+    this.apiSecret = API_SECRET;
```

Comment:
Is there any value to putting the global values into private fields? It seems like it would be easier to debug and inspect code if we just used the globals directly where needed.

Reply:
you are right, approved this approach.